.github/workflows/ci_cd_night.yml

```
@@ -22,7 +21,7 @@ jobs:
     name: "Doc build"
     runs-on: ubuntu-latest
     steps:
-      - uses: ansys/actions/doc-build@main
+      - uses: ansys/actions/doc-build@feat/check-actions-security-action
```

```suggestion
      - uses: ansys/actions/doc-build@main
```
Needs to be reverted before merging